### PR TITLE
refactor(entity): add identifiedBy on all ressources

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Adds identifiedBy uri on all ressources.
* Translates subject to language change.

### CODE REVIEW

**Review by rerowep — APPROVED** [review]

**Review by PascalRepond — APPROVED** [review]

File: projects/admin/src/app/record/detail-view/entities-detail-view/remote/remote-organisation-detail-view/remote-entities-organisation-detail-view.component.html [hunk 1]
```diff
@@ -15,59 +15,46 @@
   You should have received a copy of the GNU Affero General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<dl class="entity-fields">
```
Comment (jma) [inline projects/admin/src/app/record/detail-view/entities-detail-view/remote/remote-organisation-detail-view/remote-entities-organisation-detail-view.component.html:18]: Why did you remove the `<dl></dl>`?

File: projects/admin/src/app/record/detail-view/entities-detail-view/remote/remote-person-detail-view/remote-entities-person-detail-view.component.html [hunk 2]
```diff
@@ -15,68 +15,55 @@
   You should have received a copy of the GNU Affero General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<dl class="entity-fields">
```
Comment (jma) [inline projects/admin/src/app/record/detail-view/entities-detail-view/remote/remote-person-detail-view/remote-entities-person-detail-view.component.html:18]: same as previous

**Review by jma — APPROVED** [review]
